documentation/src/main/asciidoc/mapping.asciidoc

```diff
@@ -1497,6 +1497,16 @@ prepareMetadata() and set(String, Object, Document, LuceneOptions, Metadata meta
 allows to add additional metadata to the parsing process (for example the file name) and the latter
 allows you to index metadata discovered during the parsing process.
 
+[NOTE]
+.Sortable fields with Tika
+====
+If you want to add multiple fields in your TikaMetadataProcessor, and also want to make those
+fields sortable, you should make your processor implement the TikaFieldMetadataProvidingMetadataProcessor.
```

Comment:
OK. Yoann, could you simply add backticks to the class names so that they are formatted nicely in the doc? Once it's done, I'll merge this one.